common/templates/shareCitation.modal.html

```diff
@@ -7,6 +7,16 @@ <h2 class="modal-title text-center"><strong>Share</strong>
     <ul>
         <li id="share-link">
             <h2><b>Share Link</b></h2>
+            <div ng-if="ctrl.versionLink">
+                <h3>
+                    <b>Data Snapshotted
+                        <span ng-if="ctrl.moreThanWeek()"> at </span>
+                        <span tooltip-placement="bottom" ng-attr-uib-tooltip="{{!ctrl.moreThanWeek() ? 'Data snapshotted at ' + ctrl.versionDate : undefined}}">{{ctrl.versionDateRelative}}</span>
```

Comment:
If the timestamp is more than a week old, we're going to show only the date. So I feel like we can always show the tooltip. It's always going to provide more information.

Reply:
In the case that it's more than a week old, the exact date will be shown with the same text as the tooltip. I think the only extra information would be `hh:mm:ss`.